.gitignore

```diff
@@ -54,3 +54,4 @@ risedev-compose.yml
 ft.txt
 # generated output from regress tests
 src/tests/regress/output/*
+/src/tests/sync/slt/e2e_test/
```

Comment:
Is this required?

Reply:
For specific tests, we copy some slt files to this temp folder.
Fixed.